guides/v2.2/howdoi/checkout/checkout-add-custom-carrier.md

```diff
@@ -0,0 +1,299 @@
+---
```

Comment:
See requested changes for 2.1 version and apply in this version.

Also, it seems that v2.1, v2.2, and v2.3 should not be separate files. If the content is the same, keep one and symlink to the rest, so there are not three versions of the same file.

Reply:
Hi, @shrielenee !

Thank you for reviewing! 

> If the content is the same, keep one and symlink to the rest,....

Content for 2.1, 2.2, and 2.3 is not the same. The difference is in the content of `app/code/Vendor/CustomShipping/composer.json`.

Reply:
Ok. That makes sense then. Please take the suggested revisions I made in v2.1 and apply them as applicable to v2.2 and v2.3 files.

Reply:
Make the suggested revisions, per the comments I added in v2.1 page, to v2.2 and v2.3 pages where applicable.